### ISSUE TITLE
Not playing nicely with NetworkManager 1.0.12-1

### ISSUE THREAD

**Issue by stuartpb** [gnab/rtl8812au#81]
Users have been reporting issues with NetworkManager not recognizing this family of wireless adapters after upgrading to NetworkManager 1.0.12-1:

https://bbs.archlinux.org/viewtopic.php?id=210868

Arch Linux Forum user V1del [hypothesizes](https://bbs.archlinux.org/viewtopic.php?pid=1617699#p1617699) that this issue is being caused by the drivers not putting the devices into the correct management state, which conflicts with [a patch introduced to the NetworkManager source tree in February](https://cgit.freedesktop.org/NetworkManager/NetworkManager/commit/?h=1.0.12&id=a3c0166a2064c41408d1e08defee3a7e4a0ce3f7).

See also lwfinger/rtl8188eu#171

**Comment by abc-mikey** [comment]
Yep, I seem to be seeing this issue and can reverse the regression with a downgrade of network manager.
